frontend/javascripts/navbar.tsx

```diff
@@ -162,20 +162,7 @@ function UserInitials({
       </Avatar>
 
       {isMultiMember ? (
-        <SwapOutlined
-          style={{
-            position: "absolute",
-            top: 2,
-            right: -5,
-            marginRight: 0,
-            minWidth: 12,
-            height: 12,
-            lineHeight: "12px",
-            fontSize: 12,
-            color: "#75df4a",
-          }}
-          title="You are member of multiple organizations. Click the avatar to switch between them."
-        />
+        <SwapOutlined title="You are member of multiple organizations. Click the avatar to switch between them." />
```

Comment:
thanks for the extraction! one last thing: could you add a `className="swap-account-icon"` (or another name) and use that class name in the less file? the current way would also change other swap-icons if they are used (now or in the future).